server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectSessionService.java

```diff
@@ -273,34 +294,17 @@ public ConnectionSession create(@NotNull CreateSessionReq req) {
         // TODO: query from use config service
         DefaultConnectSessionFactory sessionFactory = new DefaultConnectSessionFactory(
                 connection, getAutoCommit(connection), factory);
-        DefaultConnectSessionIdGenerator idGenerator = new DefaultConnectSessionIdGenerator();
-        idGenerator.setDatabaseId(req.getDbId());
-        idGenerator.setFixRealId(StringUtils.isBlank(req.getRealId()) ? null : req.getRealId());
-        if (Objects.nonNull(req.getFrom()) && isOBCloudEnvironment()) {
-            idGenerator.setHost(req.getFrom());
-        } else {
-            idGenerator.setHost(stateHostGenerator.getHost());
-        }
-        sessionFactory.setIdGenerator(idGenerator);
-        long timeoutMillis = TimeUnit.MILLISECONDS.convert(sessionProperties.getTimeoutMins(), TimeUnit.MINUTES);
-        timeoutMillis = timeoutMillis + this.connectionSessionManager.getScanIntervalMillis();
-        sessionFactory.setSessionTimeoutMillis(timeoutMillis);
-        ConnectionSession session = connectionSessionManager.start(sessionFactory);
-        if (session == null) {
-            throw new BadRequestException("Failed to create a session");
-        }
-        try {
-            initSession(session, connection);
-            log.info("Connect session created, connectionId={}, session={}", dataSourceId, session);
-            return session;
-        } catch (Exception e) {
-            log.warn("Failed to create a session", e);
-            if (sessionProperties.getSingleSessionMaxCount() >= 0) {
-                limitService.decrementSessionCount(authenticationFacade.currentUserIdStr());
-            }
-            session.expire();
-            throw e;
-        }
+        sessionFactory.setIdGenerator(getIdGenerator(req));
```

Comment:
DefaultConnectSessionFactory should be include in the method `startConnectionSession`
cause that the `LogicalConnectionSessionFactory` is included in the method `createLogicalSession`

Reply:
`startConnectionSession` does not distinguish logical or default connection session. It also calls `startConnectionSession` in `createLogicalSession`.